### PR TITLE
Added Linux Workload to Migrate & Modernize Microhack

### PR DESCRIPTION
**Pull request by ASchwarz1984**
Added Linux Workload to Migrate & Modernize MicroHack.
Removed the Linux Migration Microhack as part of the update.

### CODE REVIEW

File: 03-Azure/01-03-Infrastructure/06_Migration_Datacenter_Modernization/resources/source.bicep [hunk 1]
```diff
@@ -325,15 +321,22 @@ resource vm2 'Microsoft.Compute/virtualMachines@2022-03-01' = {
...
     storageProfile: {
       imageReference: {
-        publisher: 'MicrosoftWindowsServer'
-        offer: 'WindowsServer'
-        sku: '2022-datacenter-smalldisk-g2'
+        publisher: 'RedHat'
+        offer: 'RHEL'
+        // sku: '81-ci-gen2' https://github.com/MicrosoftDocs/azure-docs/issues/84430
+        // sku: '8-gen2'
+        // sku: '7_9'
```
Comment (cthoenes) [inline 03-Azure/01-03-Infrastructure/06_Migration_Datacenter_Modernization/resources/source.bicep:338]: @ASchwarz1984 can we clean up the nun used SKUs?

**Review by cthoenes — COMMENTED** [review]
Hi @ASchwarz1984  2 small requests.

File: 03-Azure/01-03-Infrastructure/06_Migration_Datacenter_Modernization/walkthrough/challenge-5/img/mal1.png
[the remark this answers is not in the document]
Comment (ASchwarz1984) [reply 03-Azure/01-03-Infrastructure/06_Migration_Datacenter_Modernization/walkthrough/challenge-5/img/mal1.png:1]: Yes, the old picture was not used.

File: 03-Azure/01-03-Infrastructure/06_Migration_Datacenter_Modernization/resources/source.bicep (hunk 1 above)
Comment (ASchwarz1984) [reply 03-Azure/01-03-Infrastructure/06_Migration_Datacenter_Modernization/resources/source.bicep]: done

**Review by ASchwarz1984 — COMMENTED** [review]
Fixed

**Review by cthoenes — APPROVED** [review]
thank you

**Pull request merged**